llvm/lib/Target/PowerPC/PPC.td

```diff
@@ -577,79 +578,82 @@ include "GISel/PPCRegisterBanks.td"
 //
 
 def : Processor<"generic", G3Itineraries, [Directive32, FeatureHardFloat,
-                                           FeatureMFTB]>;
+                                           FeatureMFTB, FeatureCRBits]>;
```

Comment:
This next few lines below basically mean the following processors contain CR bits (but won't be on, because we turn them on for P8+), right? Did we use a particular way to verify for certain that these processors have CR bit support?